e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/add-to-an-existing-payment.spec.js

```diff
@@ -55,18 +55,22 @@ context(`${PDC_TEAMS.PDC_RECONCILE} users can add fee records to existing paymen
     cy.task(NODE_TASKS.REINSERT_ZERO_THRESHOLD_PAYMENT_MATCHING_TOLERANCES);
   });
 
+  afterEach(() => {
+    cy.task(NODE_TASKS.REMOVE_ALL_UTILISATION_REPORTS_FROM_DB);
+    cy.task(NODE_TASKS.REMOVE_ALL_PAYMENTS_FROM_DB);
+  });
+
   const navigateToAddToExistingPaymentScreenForFirstFeeRecord = () => {
     pages.landingPage.visit();
     cy.login(USERS.PDC_RECONCILE);
 
     cy.visit(`utilisation-reports/${REPORT_ID}`);
-    cy.get(`[type="checkbox"][id="feeRecordIds-${FEE_RECORD_ID_ONE}-reportedPaymentsCurrency-${PAYMENT_CURRENCY}-status-${FEE_RECORD_STATUS.TO_DO}"]`).check();
-    cy.get('[type="submit"]').contains('Add a payment').click();
+    pages.utilisationReportPage.premiumPaymentsTab.premiumPaymentsTable.getCheckbox([FEE_RECORD_ID_ONE], PAYMENT_CURRENCY, FEE_RECORD_STATUS.TO_DO).check();
 
+    cy.get('[data-cy="add-a-payment-button"]').contains('Add a payment').click();
     cy.url().should('eq', relative(`/utilisation-reports/${REPORT_ID}/add-payment`));
 
-    cy.get('[type="submit"]').contains('Add reported fee to an existing payment').click();
-
+    cy.get('[data-cy="add-fees-to-an-existing-payment-button"]').contains('Add reported fee to an existing payment').click();
```

Comment:
Since we're using a data-cy (nice), we shouldn't need `contains`, simpler. Can we remove these please? Same on line 70

Reply:
I'm not sure about existing patterns around reports etc - i'd usually lean towards not having `cy.get()` in the E2E specs and instead have these in a `page` definition or partials, if used in more than one place

So then the E2E test is just e.g `addFeesToExistingPaymentButton.click();`

Also makes it easier for future updates, if the button or selector changes, can just be changed in one place